### PR TITLE
Season4 teamtables

### PR DESCRIPTION
**Pull request by Raphib737**
moved tables to its own component to remove duplication, but lost the styling for it. Have not done anything with the typography and children section in line 205 yet.

### CODE REVIEW

File: src/components/TeamGeneratorBeta.tsx [hunk 1]
```diff
@@ -40,20 +40,40 @@ const useStyles = makeStyles({
   dropdownButton:{width:'100%'}
 });
 
+type TeamStats = {
```
Comment (michaelgerakis) [inline src/components/TeamGeneratorBeta.tsx]: Can we call this TeamTableProps?

File: src/components/TeamGeneratorBeta.tsx [hunk 2]
```diff
@@ -40,20 +40,40 @@ const useStyles = makeStyles({
...
+  color: string,
+  summoners: string[],
+  winRate: number
+}
+
+export const TeamTables = ({ color, summoners, winRate}: TeamStats)=>
```
Comment (michaelgerakis) [inline src/components/TeamGeneratorBeta.tsx]: nit: Can we call this TeamTable instead of TeamTables

**Review by michaelgerakis — CHANGES_REQUESTED** [review]
Please run the formatter over this file! Here's the command:

prettier --write "src/**/*.{ts,tsx}"

File: src/components/TeamGeneratorBeta.tsx (hunk 2 above)
Comment (Raphib737) [reply src/components/TeamGeneratorBeta.tsx]: yeah that makes more sense

File: src/components/TeamGeneratorBeta.tsx (hunk 1 above)
Comment (Raphib737) [reply src/components/TeamGeneratorBeta.tsx]: Yeet

**Review by michaelgerakis — APPROVED** [review]

**Pull request merged**